playbooks/common/openshift-etcd/scaleup.yml

```diff
@@ -53,3 +56,25 @@
     retries: 3
     delay: 30
     until: scaleup_health.rc == 0
+
+- name: Update master etcd client urls
+  hosts: oo_masters_to_config
```

Comment:
Make it serial so only one master is restarted at one time?

Reply:
Done!